packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -531,8 +368,64 @@ function joinNonEmptyValues( joinString, ...values ) {
 	return values.filter( value => value.length > 0 ).join( joinString );
 }
 
-const StateSelectWrapper = styled.div`
-	& select {
-		width: 100%;
+function contactDetailsFormat( { isDomainFieldsVisible } ) {
+	if ( isDomainFieldsVisible ) {
+		return 'DOMAINS';
 	}
-`;
+	return 'DEFAULT';
+}
+
+function renderContactDetails( {
+	translate,
+	domainName,
+	isDomainFieldsVisible,
+	contactInfo,
+	renderDomainContactFields,
+	setters,
+	PhoneInput,
+	CountrySelectMenu,
+	countriesList,
+} ) {
+	const format = contactDetailsFormat( { isDomainFieldsVisible } );
```

Comment:
If there's only one argument, we probably can pass it without an object wrapper and destructuring.